CRM/Activity/ActionMapping.php

```diff
@@ -30,7 +30,6 @@
  * @copyright CiviCRM LLC (c) 2004-2019
  */
 
-use Civi\ActionSchedule\RecipientBuilder;
```

Comment:
@totten this is an interesting one

Reply:
Agree. I opened that file to double-check references, and it seemed OK without the `use` statement. Also, `CRM_Activity_ActionMappingTest`, has coverage of this file.